templates/btax/header.html

```diff
@@ -0,0 +1,32 @@
+{% load staticfiles %}
```

Comment:
this header is identical to the standard taxbrain header, is it not? Shouldn't we just import that template instead?